CODING_STYLE.md

```diff
@@ -26,7 +26,7 @@ line width, indentation and ordering (for includes, using directives and etc). 
 * Use **snake_case** for namespace names and build targets
 * Use **UPPER_SNAKE_CASE** for macros.
 * Use **kPascalCase** for static constants and enumerators.
-
+* Use **TEST_** prefix for test only class methods, e.g. obj.TEST_foo().
```

Comment:
This makes sense, but do we have examples of test-only class methods?

Reply:
Yeah, I'm not sure what test only class methods means. Are these methods that are added to classes for exclusive use in tests ? Would something like PlanBuilder or QueryAssertions be considered test only ?